src/app/src/components/EmbeddedMapThemeConfig.jsx

```diff
@@ -50,52 +53,103 @@ const fontSelectStyles = {
     }),
 };
 
-function EmbeddedMapThemeConfig({ color, setColor, font, setFont, errors }) {
+function EmbeddedMapThemeConfig({
+    color,
+    setColor,
+    font,
+    setFont,
+    errors,
+    embedLevel,
+}) {
     const value = fontOptions.find(f => font === f.value) || OARFont;
     return (
         <div style={styles.section}>
             <Typography style={styles.sectionHeader}>Theme</Typography>
             <div style={styles.subsection}>
                 <Typography style={styles.subsectionHeader}>Color</Typography>
-                <Typography>
-                    Use a color with sufficient contrast against white
-                    backgrounds & labels.
-                </Typography>
-                {errors?.color && (
-                    <Typography style={{ color: 'red' }}>
-                        Error: {errors.color.join(', ')}
-                    </Typography>
+                {embedLevel === 1 ? (
+                    <>
+                        <Typography>
+                            Choose the primary color for the app.
```

Comment:
Could use some input here on what this should say.

Reply:
Changing "app" → "embedded map" sounds good to me